test/start.test.js

```diff
@@ -558,6 +558,35 @@ test('should start the server with watch options that the child process restart
   t.pass('should receive restart event')
 })
 
+test('should start the server with watch and verbose-watch options that the child process restart when directory changed with console message about changes ', { skip: onGithubAction }, async (t) => {
```

Comment:
Why skipping with onGithubAction?

Reply:
I used the same params of the test [at line 532](https://github.com/fastify/fastify-cli/blob/master/test/start.test.js#L532)